### PR TITLE
Gleam for PHP users

### PR DESCRIPTION
**Pull request by inoas**
I am looking for a review

### CODE REVIEW

**Review by lpil — COMMENTED** [review]
Brilliant stuff! Fantastic work! I have asked one question

File: cheatsheets/gleam-for-php-users.md [hunk 1]
```diff
@@ -0,0 +1,1290 @@
...
+}
+```
+
+## Modules
+
+### PHP
+
+PHP does not feature modules, but many other containers such as classes, traits and interfaces.
+Historically a single file can contain many classes, traits and interfaces one after another,
+though this is commonly not the case.
```
[the remark this answers is not in the document]
Comment (inoas) [reply cheatsheets/gleam-for-php-users.md:1141]: PHP namespaces are just that, namespaces. They do not enforce or require to be mapping to any kind of files. A namespace is - imho - maybe best described as a virutal address like an internet domain or a linux file system path.

You can put all kinds of things in there (classes, interfaces). They can be relative (no slash in front) or absolute (slash in front).

See <https://www.php.net/manual/en/language.namespaces.basics.php>

So in practise they are somewhat similar to Elixir modules, which can have any path as a module name which does not need to map to the directory/file structure. But they are less similar to Gleam modules which are (luckily) much more restrictive.

It is good practise in PHP, map the file system hierarchy to the namespaces and vice-versa though (just not required at all).

[the remark this answers is not in the document]
Comment (inoas) [reply cheatsheets/gleam-for-php-users.md]: I feel like static classes are more like gleam modules.

within classes that only have static members (static classes):

- gleam consts map to php class consts or static members
- gleam functions map to php functions
- gleam's main function of a module exists as a global scope that is being executed when the static class file is being required in PHP
- there is no direct equivalent for custom types within modules though something could be constructed that lives in a sub namespace 

Generally the concepts do not map 1 to 1 but are just similar. I will update this PR.

[the remark this answers is not in the document]
Comment (inoas) [reply cheatsheets/gleam-for-php-users.md]: Or to turn it around. Gleam modules are like a combination of PHP namespaces and PHP static classes. I have updated the PR. PHP namespaces alone could just place some const or function in the namespace hierarchy say `Foo/bar`. In Gleam that is simply not possible, everything is nested in modules.

**Review by lpil — APPROVED** [review]
Incredible work, thank you!

**Pull request merged**